extensions/npm/main.ts

```diff
@@ -94,19 +103,33 @@ if (Deno.args.length >= 1
         console.log(`[${green("phylum")}] Packages built successfully.`);
     }
 } else {
-    let status = await Deno.run({ cmd: ['npm', ...Deno.args] }).status();
+    let status = PhylumApi.runSandboxed({
+        cmd: 'npm', 
+        args: Deno.args,
+        exceptions: {
+            write: ['~/.npm', './'],
+            read: ['~/.npm', './'],
+            run: ['npm'],
+            net: true,
+        }
+    });
```

Comment:
I'd honestly be surprised if these were all the permissions required to run *any* `npm` command. But I guess we'll find out?

Reply:
I'd be surprised too, but as a developer I also would not want `npm` to go and modify files in unpredictable places. I'd rather find out and evaluate more exceptions as we go :D